lib/core/app/app_router.dart

```diff
@@ -190,6 +199,7 @@ class AppRouter {
                 .read<OrganisationDetailsCubit>()
                 .getOrganisationDetails(mediumID);
 
+            //TODO: Ask Daniela why not startDeepLinkCoinFlow instead?
```

Comment:
Does this need to be resolved?

Reply:
@Daniela510 can you recall something re this?